src/components/aboutCollection/AboutCollection.js

```diff
@@ -0,0 +1,24 @@
+import collapseArrow from "assets/newCollection/arrows/collapse.png";
+import expandArrow from "assets/newCollection/arrows/expand.png";
+import styles from "components/aboutCollection/aboutCollection.module.css";
+import { useState } from "react";
```

Comment:
importy z node modules powinny być imo na samej górze, dodaj na to jakąś regule w eslint